config/yurtadm/cloudimage/openyurt-latest/install-openyurt.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Copyright 2022 The OpenYurt Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "[INFO] Start installing OpenYurt."
+
+## label node
+kubectl label node $HOSTNAME openyurt.io/is-edge-worker=false
+
+## install openyurt components
+kubectl apply -f manifests/yurt-controller-manager.yaml
+kubectl apply -f manifests/yurt-tunnel-agent.yaml
+kubectl apply -f manifests/yurt-tunnel-server.yaml
+kubectl apply -f manifests/yurt-app-manager.yaml
+kubectl apply -f manifests/yurthub-cfg.yaml
+
+## configure coredns
+kubectl apply -f manifests/coredns.yaml
+kubectl annotate svc kube-dns -n kube-system openyurt.io/topologyKeys='openyurt.io/nodepool'
+kubectl scale --replicas=0 deployment/coredns -n kube-system
```

Comment:
maybe it's more reasonable to scale deployment/coredns before applying daemonset/coredns.

Reply:
ok